src/nvidia_resiliency_ext/inprocess/abort.py

```
@@ -136,27 +136,28 @@ def _check_fr_env():
                 except ValueError:
                     env_value_int = 0
 
```

Line 134-138 can be deleted.

---

env var can be a value not convertible to int

---

We should flag the error to the user as it is a config error. Better than suppressing the error and quietly using 0 underneath.